### PR TITLE
#108 added missing fields to items object

### PR DESCRIPTION
**Pull request by Sobert**
Hopefully I understood correctly what was needed.

### CODE REVIEW

**Comment by wafflespeanut** [comment]
Please add a comment in the issue saying you're willing to work on it, so that I can assign it to you and we can avoid duplicate efforts for someone else.

File: core/src/v2/models.rs [hunk 1]
```diff
@@ -332,6 +332,28 @@ pub struct Items {
...
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub maximum: Option<f32>,
+    #[serde(rename = "exclusiveMaximum", skip_serializing_if = "Option::is_none")]
+    pub exclusive_maximum: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub minimum: Option<f32>,
+    #[serde(rename = "exclusiveMinimum", skip_serializing_if = "Option::is_none")]
+    pub exclusive_minimum: Option<bool>,
+    #[serde(rename = "maxLength", skip_serializing_if = "Option::is_none")]
+    pub max_length: Option<i32>,
```
Comment (wafflespeanut) [inline core/src/v2/models.rs]: Even though the spec says this is an integer, it also says that the integer must be greater than or equal to zero, in which case, we should go for the unsigned version (`u32`).

File: core/src/v2/models.rs [hunk 2]
```diff
@@ -332,6 +332,28 @@ pub struct Items {
...
+    #[serde(rename = "minLength", skip_serializing_if = "Option::is_none")]
+    pub min_length: Option<i32>,
```
Comment (wafflespeanut) [inline core/src/v2/models.rs]: Same as `max_length`.

File: core/src/v2/models.rs [hunk 3]
```diff
@@ -332,6 +332,28 @@ pub struct Items {
...
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub pattern: Option<String>,
+    #[serde(rename = "maxItems", skip_serializing_if = "Option::is_none")]
+    pub max_items: Option<i32>,
```
Comment (wafflespeanut) [inline core/src/v2/models.rs]: Same as the above.

File: core/src/v2/models.rs [hunk 4]
```diff
@@ -332,6 +332,28 @@ pub struct Items {
...
+    #[serde(rename = "minItems", skip_serializing_if = "Option::is_none")]
+    pub min_items: Option<i32>,
```
Comment (wafflespeanut) [inline core/src/v2/models.rs]: Same as the above.

**Comment by Sobert** [comment]
> Please squash all your commits once you've made the changes. :)

It should be all good now :)

**Comment by wafflespeanut** [comment]
Thanks!

**Pull request merged**